src/pq/connection.cr

```diff
@@ -54,6 +54,18 @@ module PQ
       if serv_ssl
         ctx = OpenSSL::SSL::Context::Client.new
         ctx.verify_mode = OpenSSL::SSL::VerifyMode::NONE # currently emulating sslmode 'require' not verify_ca or verify_full
+        sslcert = @conninfo.sslcert
+        if sslcert
+          ctx.certificate_chain = sslcert
+        end
+        sslkey = @conninfo.sslkey
+        if sslkey
+          ctx.private_key = sslkey
+        end
+        sslrootcert = @conninfo.sslrootcert
+        if sslrootcert
+          ctx.ca_certificates = sslrootcert
+        end
```

Comment:
This can be a little more concise if they read something like 
```crystal 
ctx.ca_certificates = @conninfo.sslrootcert if @conninfo.sslrootcert
```

I haven't done much with cert auth myself, so I don’t know if all 3 are always required. If so, it could instead be one if block that sets all 3 rather than 3 separate conditionals.

Reply:
New to Crystal so still looking of the right way to formulate code in an idiomatic way.

Reply:
Changed the code a bit to remove the extra assignment lines.